packages/wrangler/src/__tests__/pubsub.test.ts

```diff
@@ -83,15 +80,30 @@ describe("wrangler", () => {
 			describe("create", () => {
 				function mockCreateRequest(expectedNamespaceName: string) {
 					const requests = { count: 0 };
-					setMockResponse(
-						"/accounts/:accountId/pubsub/namespaces",
-						"POST",
-						([_url, accountId], { body }) => {
-							expect(accountId).toEqual("some-account-id");
-							const namespaceName = JSON.parse(body as string).name;
-							expect(namespaceName).toEqual(expectedNamespaceName);
-							requests.count += 1;
-						}
+					msw.use(
+						rest.post(
+							"*/accounts/:accountId/pubsub/namespaces",
+							async (req, res, ctx) => {
+								expect(req.params.accountId).toEqual("some-account-id");
+								const namespace = await req.json();
+								expect(namespace.name).toEqual(expectedNamespaceName);
+								requests.count++;
+								return res.once(
+									ctx.status(200),
+									ctx.json({
+										success: true,
+										errors: [],
+										messages: [],
+										result: {
+											id: "some-namespace-id",
+											name: namespace.name,
+											created_at: "3005-01-01T00:00:00.000000Z",
+											updated_at: "3005-01-01T00:00:00.000000Z",
+										},
+									})
+								);
+							}
+						)
```

Comment:
This looks like a success response -- is there a reason this isn't kind of in its own file with the other handlers?

Reply:
Ah ok I see, because we count requests. Yeah sure, seems good to me.